app/src/main/res/values/strings.xml

```diff
@@ -69,6 +69,9 @@
   <string name="stripe_publishable_key">pk_live_zjuK52lEUYcvBhIXEUnOEJzk</string>
   <string name="stripe_version" tools:ignore="TypographyDashes">2015-08-07</string>
 
+  <!-- Comments -->
+  <string name="this_comment_is_under_review">This comment is under review.</string>
```

Comment:
minor suggestion, in order to identify quickly the string being a placeholder until getting translations, we could name the string like name="FPO_this_comment_is_under_review"

Reply:
sounds good to me, i'll update